src/authStrategies/LocalAuth.js

```diff
@@ -44,7 +44,11 @@ class LocalAuth extends BaseAuthStrategy {
 
     async logout() {
         if (this.userDataDir) {
-            return (fs.rmSync ? fs.rmSync : fs.rmdirSync).call(this, this.userDataDir, { recursive: true, force: true });
+            await fs.promises.rm(this.userDataDir, { recursive: true, force: true })
+                .catch(() => {
+                    return this.logout();
```

Comment:
Catch should just give-up right? Won't this run forever if something breaks on the promise ?